packages/clerk-js/src/ui/components/OrganizationProfile/ProfileSettingsPage.tsx

```diff
@@ -60,6 +68,20 @@ export const ProfileSettingsPage = withCardStateProvider(() => {
       .catch(err => handleError(err, [], card.setError));
   };
 
+  const onChangeSlug = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value && !isValidSlug(event.target.value)) {
```

Comment:
Are we keeping the FE validation here? 
I thought that we'd simply show the FAPI payload!

Reply:
@nikosdouvlis I thought we were keeping this, no?